src/ProductConstructionService/Readme.md

```diff
@@ -1,6 +1,10 @@
 # Starting the service locally
 
-To run the Product Construction Service locally, set the `ProductConstructionService.AppHost` as Startup Project, and run with F5
+To run the Product Construction Service locally, set the `ProductConstructionService.AppHost` as Startup Project, and run with F5.
+
+When running locally, the service will attempt to read secrets from the `ProductConstructionDev` KeyVault, using your Microsoft credentials. If you're having some authentication double check the following:
```

Comment:
Can you make the KV name a link to Azure?

Reply:
Added, just need to make sure to update the url when we move to westus